tests/copyright/check.sh

```diff
@@ -377,7 +378,7 @@ is_balabit_copyright() {
  grep --quiet --extended-regexp "\
 ^\
 (Copyright \(c\) ([0-9, -]+) [^ <][^<]*<br>)*\
-Copyright \(c\) ([0-9, -]+) (Bala[bB]it)|(One Identity)<br>\
+Copyright \(c\) ([0-9, -]+) (Bala[bB]it)|((One Identity)(, LLC\.)?)<br>\
```

Comment:
OI's legal recommendation is to use `One Identity LLC.` without the comma.
I think we should conform to that form.

An open-source example:
https://github.com/google/oss-fuzz/blob/master/projects/digest/fuzz_digest.py#L2